pkg/graveler/graveler.go

```diff
@@ -1029,10 +1029,10 @@ type RefManager interface {
 	// RemoveCommit deletes commit from store - used for repository cleanup
 	RemoveCommit(ctx context.Context, repository *RepositoryRecord, commitID CommitID) error
 
-	// FindMergeBase returns the merge-base for the given CommitIDs
+	// FindMergeBase returns the merge-base commit and its CommitID for the given CommitIDs
```

Comment:
1. The code does not appear to return the commit-ID of the merge-base.
1. "Key, value" is much more common than "value, key", so I prefer to return ID _before_ the commit.